lib/tmuxinator/project.rb

```diff
@@ -56,14 +79,6 @@ def pre_window
       end
     end
 
-    def attach
-      attach = true
-      if !yaml["attach"].nil?
```

Comment:
@deivid-rodriguez Can `yaml["attach"]` be `false`? Then this will not work as expected.